service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -244,6 +249,41 @@ internal class RevurderingServiceImpl(
         }
     }
 
+    override fun forhåndsvarsleEllerSendTilAttestering(
```

Comment:
Noen spesiell grunn til at dette ikke kunne være to distinkte operasjoner?

Reply:
Ja. Vi prøvde dem ut som 2 endepunker. Det førte til at revurderingen får en state-endring som rendret neste steget for saksbehandler før han ble navigert til saksoversikten. Det føltes ganske useriøst ut, så vi endte opp med denne.